.github/workflows/OWNERS

```diff
@@ -0,0 +1,5 @@
+filters:
+  ".\\e2e-ci-ppc64le.yml":
+    approvers:
+      - snehakpersistent
+      - mayurwaghmode
```

Comment:
@stevekuznetsov @ncdc this could be a compromise. Mayur and Sneha own this test suite. It will be optional for PRs. They maintain it and have to follow the other test jobs when we change them. Maybe an SLA like "if it fails for longer than 4 weeks (1 sprint) we take the liberty to disable or remove it" ?

Reply:
I'm ok with that.